modules/serverless-functions-secrets-configmaps-manually.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-accessing-secrets-configmaps.adoc
+
+[id="serverless-functions-secrets-configmaps-manually_{context}"]
+= Adding the configuration manually
+
+.Procedure
+
+. Open the `func.yaml` file for your function.
+
+. For every secret you want to mount as a volume, add this to the `volumes` section:
++
+[source,yaml]
+----
+- secret: mysecret
+  path: /workspace/secret
+----
```

Comment:
I'd show the rest of the file to give more context rather than just the snippet, this doesn't show the `volumes` bit that's referred to or where in the file this appears